shap/explainers/_explainer.py

```diff
@@ -6,7 +6,7 @@
 
 
 class Explainer():
-    def __init__(self, model, masker=None, link=links.identity, algorithm="auto", output_names=None):
+    def __init__(self, model, masker=None, link=links.identity, algorithm="auto", output_names=None, fixed_context = 1):
```

Comment:
We don't need fixed_context until explain_row right? So it would probably better be a parameter of the __call__ method just for the Partition class.

Reply:
I have updated the files and sent a new PR. Request you to please take a look.